docs/manifest.json

```diff
@@ -197,6 +197,42 @@
 		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/developers/tutorials/block-tutorial/generate-blocks-with-wp-cli.md",
 		"parent": "block-tutorial"
 	},
+	{
+		"title": "Getting Started with JavaScript",
+		"slug": "javascript",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/developers/tutorials/javascript/readme.md",
+		"parent": "tutorials"
+	},
+	{
+		"title": "PluginsBackground",
```

Comment:
@mkaz just noticed the titles in the manifest are incorrect. Did you generate the manifest automatically? Could be an issue in the generator if so, which should get fixed before we merge…

Reply:
I did generate the manifest using `npm run docs:build`

Reply:
Ah, fixed. 

I generated the manifest prior to fixing the header tags and didn't regenerate after. I just ran the script and committed the updated manifest, which has the correct titles. 👍